patterns/jolla-hw-adaptation-discovery.yaml

```diff
@@ -40,14 +40,9 @@ Requires:
 - qt5-qpa-hwcomposer-plugin
 - qtscenegraph-adaptation
 
-# Add GStreamer v1.0 as standard
-- gstreamer1.0
-- gstreamer1.0-plugins-good
-- gstreamer1.0-plugins-base
-- gstreamer1.0-plugins-bad
-- nemo-gstreamer1.0-interfaces
```

Comment:
Are all these above still needed and will they be pulled in automatically? If yes, LGTM.

Reply:
They were moved to patterns-sailfish-mw late last year. I only just merged the removal of those to dhc yesterday, to make sure everyone had time to catch up first.